guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -20,14 +20,16 @@ An internal label that matches the value is also created and cannot be changed a
 If you do not specify a value, an organization called *Default Organization* with the label *Default_Organization* is created.
 You can rename the organization name but not the label.
 
-* By default, all configuration files configured by the installer are managed by Puppet.
-When `{foreman-installer}` runs, it overwrites any manual changes to the Puppet managed files with the initial values.
-By default, {ProjectServer} is installed with the Puppet agent running as a service.
-If required, you can disable Puppet agent on {ProjectServer} using the `--puppet-runmode=none` option.
-
-* If you want to manage DNS files and DHCP files manually, use the `--foreman-proxy-dns-managed=false` and `--foreman-proxy-dhcp-managed=false` options so that Puppet does not manage the files related to the respective services.
```

Comment:
Before I think about it, I've got a question: Why would the user want to manage the files manually? What is the story?

Reply:
If the installer can't write down the correct content. For example, some users want to manage multiple zone files in DNS, but the installer doesn't provide a parameter for that. Similar, with DHCP there are some more advanced configs possible that the installer doesn't have parameters for.

Reply:
Actually, I've found a whole module for this in [Satellite](https://docs.theforeman.org/nightly/Installing_Server/index-satellite.html#disabling-dns-dhcp-tftp-for-unmanaged-networks_satellite) and [Katello](https://docs.theforeman.org/nightly/Installing_Server/index-katello.html#disabling-dns-dhcp-tftp-for-unmanaged-networks_foreman).

So we can safely remove it.